conan/test_package/CMakeLists.txt

```diff
@@ -1,49 +1,70 @@
 cmake_minimum_required(VERSION 3.1)
 project(PackageTest CXX)
 
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
 find_package(userver REQUIRED)
 find_package (Python COMPONENTS Interpreter Development)
 
 add_library(${PROJECT_NAME}_objs OBJECT
     hello.hpp
     hello.cpp
 )
-target_link_libraries(${PROJECT_NAME}_objs PUBLIC userver::core)
+target_link_libraries(${PROJECT_NAME}_objs PUBLIC userver::core userver::universal)
```

Comment:
core includes the universal (universal linkage can be removed)
also, i think that it is better to link privately here. that should break only the test_core example, where
you should add userver::core linkage again

Reply:
I agree with you but it's just a test_package so I tried to make as simple as possible and universal for all targets

Reply:
ok, i can try to make this changes by myself and merge your PR tommorow